### PR TITLE
Increment Dataportal version in NIAID PROD

### PR DESCRIPTION
**Pull request by nss10**
Link to Jira ticket if there is one: [PXP-9248](https://ctds-planx.atlassian.net/browse/PXP-9248)

### Environments
- Niaid Prod

### Description of changes
- Increment Dataportal footer to include HHS responsible disclosure link

### CODE REVIEW

File: accessclinicaldata.niaid.nih.gov/manifest.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "pidgin": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pidgin:2021.09",
     "revproxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/nginx:2021.09",
     "sheepdog": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sheepdog:2021.09",
-    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:3.19.0",
+    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:3.19.1",
```
Comment (paulineribeyre) [inline accessclinicaldata.niaid.nih.gov/manifest.json:19]: was it deployed to qa-niaid? the env is down right now

Comment (nss10) [reply accessclinicaldata.niaid.nih.gov/manifest.json:19]: I've tested it beforehand. Not today. I can do it today as well if that is needed.

Comment (paulineribeyre) [reply accessclinicaldata.niaid.nih.gov/manifest.json:19]: it should be permanently deployed to QA before being deployed to prod, even if you did a temporary test in the VM https://github.com/uc-cdis/gitops-qa/blob/master/qa-niaid.planx-pla.net/manifest.json#L21

Comment (nss10) [reply accessclinicaldata.niaid.nih.gov/manifest.json:19]: Yeah! I forgot to set to set the PR -- `Ready for review` --  https://github.com/uc-cdis/gitops-qa/pull/1817

**Review by hughestr — APPROVED** [review]

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**